### PR TITLE
Updated Home Page with css, re-wrote some html blocks

### PR DESCRIPTION
**Pull request by nora-weisser**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,40 +16,40 @@
         </div>
     </div>
 
-    <div class="row registration">
-        <div class="col mentor">
+    <div class="row registration content justify-content-around">
```
Comment (dricazenck) [inline index.html:19]: as a suggestion: you can remove these new classes from here and add them into `.registration` like this: 

```
.registration {
        @extend .content;
        @extend .justify-content-around;
}
```

File: index.html [hunk 2]
```diff
@@ -16,40 +16,40 @@
...
             <p><button onclick="location.href='/mentors';" >Join as a Mentee</button></p>
         </div>
     </div>
 
     <div class="timeline">
         <h2>Timeline</h2>
-        
-        <div class="row">
+
+        <div class="row justify-content-around">
```
Comment (dricazenck) [inline index.html:52]: same here: 

```
.timeline {
   .row { 
        @extend .justify-content-around;
   }
}
```

Comment (dricazenck) [reply index.html:52]: just to make this page cleaner, but both work well.

File: index.html (hunk 1 above)
Comment (nora-weisser) [reply index.html]: fixed it!

File: index.html (hunk 2 above)
Comment (nora-weisser) [reply index.html]: cleaned this part as well

**Review by dricazenck — APPROVED** [review]

**Pull request merged**